### ISSUE TITLE
Question about identity classification head and dataset split

### ISSUE THREAD

**Issue by liubw20050702** [Zplusdragon/ReID5o_ORBench#4]
Hi, your work is really awsome! Thank you for this great work about all modalities Re-identification.

I have got a question regarding the identity classification head using during testing/training.

According to your code `ReID5o/test.py`, in line 56~59 : 
```python
    model = build_model(args, num_classes=1000)
    checkpointer = Checkpointer(model)
    checkpointer.load(f=op.join(args.output_dir, 'best.pth'))
    model.to(device)
```
the model is built with num_classes = 1000, i.e., the classification head is defined over 1000 identities.

However, in the paper (Section 2.4, Evaluation Protocol), ORBench is split into:
 - 600 identities for training
 - 400 identities for testing

So I would like to ask:
1. What is the motivation for setting the classification head to 1000 classes instead of 600?
2. During training, are only the 600 training identities supervised in the ID loss, with the remaining classes unused?
3. Would setting num_classes = 600 affect the training behavior or final performance?

In addition, I was not able to reproduce the reported results when training with 600 identities only. If possible, would you consider releasing the full training code or training scripts used in the paper? This would be very helpful for reproducibility.

Thank you again for your excellent work and for sharing the code!

**Comment by Zplusdragon** [comment]
Ideally, the number of classification heads should have been set to 600. However, due to an oversight during code implementation, it was mistakenly configured as 1000. In the actual training process, we used the 600 IDs from the training set for training. In principle, this should not significantly impact the final experimental results nor affect the main conclusions of the paper. We will shortly fix this code bug, retrain the model using the 600-class classification heads, and update the corresponding results and checkpoints in the repository.
